aries_cloudagent/multitenant/askar_profile_manager.py

```diff
@@ -60,6 +61,8 @@ async def get_wallet_profile(
             Profile: Profile for the wallet record
 
         """
+        extra_settings = extra_settings or {}
+
```

Comment:
The `extra_settings` dict is overridden later in this function. I didn't want to change anything not related to the task but it seems like this might be a mistake.